documentation/blog/2023-09-20-runes.md

```diff
@@ -143,6 +151,8 @@ export function createCounter() {
 </button>
 ```
 
+> Runes can only be used in `.svelte.js` and `.svelte.ts` outside of `.svelte` files.
```

Comment:
how about this?

```suggestion
> Outside `.svelte` components, runes can only be used in `.svelte.js` and `.svelte.ts` modules.
```

Reply:
Seems like passive speech. I remember we had a discussion last year in `svelte.dev revamp` channel about direct speech vs passive speech, direct won. 😅

I'd go with direct speech preferably.

Reply:
Both are passive voice — `runes can only be used`. Active voice would describe someone _using_ the runes, and that would mean making the actor (you) explicit rather than implicit — instead of 'runes can only be used <by you>', it would become 'you can only use runes'.

Putting the 'outside of `.svelte` files' at the end, rather than in a leading clause, introduces potential ambiguity — it could be the runes that are outside of `.svelte` files, or `.svelte.js` and `.svelte.ts`. The latter is nonsensical, but you have to spend extra time processing the sentence to realise that.

(This is what happens to a person if you spend long enough being yelled at by newsroom sub-editors.)

Reply:
I forgot ur time at NYT before making that suggestion lol 😅🤣